src/Database/Adapter/MariaDB.php

```diff
@@ -27,6 +27,10 @@ public function create(string $name): bool
     {
         $name = $this->filter($name);
 
+        if ($this->exists($name)) {
+            return true;
+        }
+
         $sql = "CREATE DATABASE IF NOT EXISTS `{$name}` /*!40100 DEFAULT CHARACTER SET utf8mb4 */;";
```

Comment:
```suggestion
        $sql = "CREATE DATABASE `{$name}` /*!40100 DEFAULT CHARACTER SET utf8mb4 */;";
```

With the new check we don't really need this.

Reply:
Also, do we have the same in other adapters?